controls/nist_ocp4.yml

```diff
@@ -1463,16 +1463,20 @@ controls:
   - high
   - moderate
 - id: AC-12(1)
-  status: pending
+  status: partial
```

Comment:
This is marked as partial becuase there's an RFE right?

Reply:
Yes, that's right